src/utilities/CustomHooks.jsx

```diff
@@ -1,6 +1,9 @@
 import axios from "axios";
 import { useDataValues } from "../contextAndReducers/DataProvider";
 import { useAuthorization } from "../contextAndReducers/AuthProvider";
+const token = () => {
```

Comment:
What is this function calling itself?

Reply:
Thanks for reviewing the pr.
This is basically getting token from the local storage